caasp-stack.yaml

```diff
@@ -70,6 +83,56 @@ parameters:
     description: Root Password for the VMs
     default: linux
 
+  cpi_auth_url: 
+    type: string
+    description: Cloud Provider Interface - keystone auth url
+    default: https://192.168.24.26:5000/
+          
+  cpi_domain_name: 
+    type: string
+    description: Cloud Provider Interface - keystone domain name
+    default: Default
+
+  cpi_tenant_name:
+    type: string
+    description: Cloud Provider Interface - keystone project
+    default: admin
+
+  cpi_region:
+    type: string
+    description: Cloud Provider Interface - keystone region
+    default: region1
+
+  cpi_username:
+    type: string
+    description: Cloud Provider Interface - keystone username
+    default: admin
+
+  cpi_password:
+    type: string
+    description: Cloud Provider Interface - keystone user password
+
+  cpi_monitor_max_retries:
+    type: number
+    description: Cloud Provider Interface - monitoring max retries
+    default: 3
+    
+  cpi_cinder_bs_version:
+    type: string
+    description: Cloud Provider Interface - cinder block storage version
+    default: auto
+    constraints:
+      - allowed_values:
+          - v1
+          - v2
+          - v3
+          - auto
+
+  cpi_ignore_volume_az:
+    type: boolean
+    description: Cloud Provider Interface - ignore volume availability zone
```

Comment:
OpenStack Cloud Provider Integration - ignore Nova and Cinder availability zone
￼